testbenches/link.mk

```diff
@@ -74,15 +74,17 @@ LDFLAGS += -lbsg_manycore_runtime -lm
 LDFLAGS += -L$(LIBRARIES_PATH) -Wl,-rpath=$(LIBRARIES_PATH)
 
 VCS_LDFLAGS    += $(foreach def,$(LDFLAGS),-LDFLAGS "$(def)")
-VCS_VFLAGS     += -M +lint=TFIPC-L -ntb_opts tb_timescale=1ps/1ps -lca -v2005 \
-                -timescale=1ps/1ps -sverilog -full64 -licqueue
+VCS_VFLAGS     += -M +lint=TFIPC -L -ntb_opts tb_timescale=1ps/1ps -lca -v2005
```

Comment:
shouldn't we be enable all lint warnings, and selectively disable some options?
e.g. +lint=all,noSVA-UA,noSVA-NSVU,noVCDE

Reply:
Yes, without a doubt.

Reply:
Though, this creates a TON of messages.

Reply:
See the branch tommy_idea_remove_irrelevant_lint_errors if you are curious